.eslintrc.cjs

```
@@ -0,0 +1,92 @@
+module.exports = {
+  env: {
+    es2022: true,
+    node: true
```

ESLint

[error] Insert `,`

(prettier/prettier)